raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -1434,7 +1436,7 @@ contract TokenNetwork is Utils {
         if (participant.balance_hash == 0 &&
             transferred_amount == 0 &&
             locked_amount == 0 &&
-            locksroot == 0
+            locksroot == keccak_of_empty
```

Comment:
What I mentioned above would apply here too.

Reply:
Yes, a buggy client might get some trouble here.  And we can also allow `locksroot == 0` and there won't be a new attack.  And maybe it' good to guide more cases into this trivial path.

I'll think for 10 minutes.

Reply:
In this case, the locksroot should be just ignored.